src/conversion/azure/azure_converter.ts

```diff
@@ -0,0 +1,83 @@
+import {NodeSpec, SymbolStore} from '../../graph';
+import {
+  AnyAzureObject,
+  ConverterStore,
+  EntityStore,
+  LocalIpConverter,
+  NetworkInterfaceConverter,
+  PublicIpConverter,
+  SubnetConverter,
+  VirtualNetworkConverter,
+} from '.';
+import {INodeSpecUniverse} from '../contracts';
+
+export class AzureConverter {
+  private readonly converters: ConverterStore;
+  private readonly entityStore: EntityStore;
+  private readonly symbolStore: SymbolStore;
+  private readonly vnetConverter: VirtualNetworkConverter;
+
+  constructor() {
+    this.entityStore = new EntityStore();
```

Comment:
I haven't decided whether I prefer to initialize members at their declaration or in the constructor. My feeling is that the idiomatic, Typescript way is to initialize at declaration because you don't need to specify the type. With that said, the jury is out for me. Do you have an opinion?

Reply:
My behavior is to default to initializing in the constructor unless there is need otherwise or a very strong culture against. 

My  primary reason for the consistency when debugging. Given a constructor it is very easy to set a break point to see when a type is created. It also means when stepping the debugger doesn't hop all around, which can be annoyingly frustrating.